frontend/src/metabase/public/components/EmbedModal/SelectEmbedTypePane/SelectEmbedTypePane.tsx

```diff
@@ -112,27 +122,46 @@ export function SelectEmbedTypePane({
         </SharingPaneButton>
 
         {/* INTERACTIVE EMBEDDING */}
-        <Link
+        <MaybeLink
           to={interactiveEmbeddingCta.url}
           target={interactiveEmbeddingCta.target}
           rel="noreferrer"
+          shouldRender={
+            !isInteractiveEmbeddingAvailable || !isInteractiveEmbeddingDisabled
+          }
+          aria-label={t`Interactive embedding`}
         >
           <SharingPaneButton
             title={t`Interactive embedding`}
             badge={<Badge color="brand">{t`Pro`}</Badge>}
             illustration={<InteractiveEmbeddingIllustration />}
+            isDisabled={isInteractiveEmbeddingDisabled}
+            disabledLink={
+              "/admin/settings/embedding-in-other-applications/full-app"
+            }
           >
             <List>
               {/* eslint-disable-next-line no-literal-metabase-strings -- only admin sees this */}
               <List.Item>{t`Embed all of Metabase in an iframe.`}</List.Item>
               <List.Item>{t`Let people can click on to explore.`}</List.Item>
               <List.Item>{t`Customize appearance with your logo, font, and colors.`}</List.Item>
             </List>
+            {!isInteractiveEmbeddingAvailable && (
```

Comment:
only for interactive embedding that has an [**unavailable** state](https://www.notion.so/metabase/Streamline-embed-flow-when-embedding-features-are-off-13869354c90180b4bc02ca29ebf61ad2?pvs=4#14c69354c90180748578f93e02f89579) (non pro). We want to lead users to learn more about the features so they could make an informed decision whether or not to buy pro. 

There was a talk to use upsell links here, but it doesn't make much sense since users still haven't understood how interactive embedding works yet, so we think it's better to send them to our embedding landing page instead of forcing them to pay.